tools/maven/checkstyle.xml

```diff
@@ -155,7 +154,7 @@ This file is based on the checkstyle file of Apache Beam.
         -->
         <module name="RegexpSinglelineJava">
             <property name="format" value="^import com.google.common.base.Preconditions;$"/>
-            <property name="message" value="Static import functions from Guava Preconditions"/>
+            <property name="message" value="Use amoro shaded guava instead."/>
         </module>
 
         <!--
```

Comment:
do you think we should add Illegal import check rules for zookeeper/thirift here or not?

Reply:
Maybe yes, we can add it in another PR.